src/app/about/page.js

```diff
@@ -15,14 +20,34 @@ const Page = () => {
         image={springPicture}
         alt="Spring 24 Picture"
       />
-      {ABOUT.map((section, index) => (
-        <div key={index} className="mb-8 text-center">
-          <Subtitle color="text-biscuits-green-200" text={section.subtitle} />
-          <div className="md:text-2xl text-base font-inika md:w-2/3 w-10/12 mx-auto">
-            {section.paragraph}
+      {ABOUT.map((section, index) => {
```

Comment:
```suggestion
      <div ref={ref} className={ANIMATION HERE}>
      {ABOUT.map((section, index) => {
```

animation is a little glitchy when scrolling, since there are multiple reference values.

Reply:
add the actual animation in the classname here, since they all perform the same animation.

you can also hard code the animate- stuff in the classname.